modules/exploits/windows/local/alpc_taskscheduler.rb

```diff
@@ -9,93 +9,64 @@ class MetasploitModule < Msf::Exploit::Local
-          'asoto-r7',             # msf module
-          'Jacob Robles'          # msf module
-        ],
-      'Platform'       => 'win',
-      'SessionTypes'   => ['meterpreter'],
-      'Targets'        =>
-        [
-          ['Windows 10 x64', { 'Arch' => ARCH_X64 }]
-        ],
-      'References'     =>
-        [
-          ['CVE', '2018-8440'],
-          ['URL', 'https://github.com/SandboxEscaper/randomrepo/'],
-        ],
-      'Notes'          =>
-        {
-          # Exploit overwrites PrintConfig.dll, which makes it unusable.
-          'Stability'   => [ OS_RESOURCE_LOSS ],
-          'Reliability' => [ REPEATABLE_SESSION ]
+    super(
+      update_info(
+        info,
+        'Name' => 'Microsoft Windows ALPC Task Scheduler Local Privilege Elevation',
+        'Description' => %q{
+          On vulnerable versions of Windows the alpc endpoint method SchRpcSetSecurity implemented
+          by the task scheduler service can be used to write arbitrary DACLs to `.job` files located
+          in `c:\windows\tasks` because the scheduler does not use impersonation when checking this
+          location. Since users can create files in the `c:\windows\tasks` folder, a hardlink can be
+          created to a file the user has read access to. After creating a hardlink, the vulnerability
+          can be triggered to set the DACL on the linked file.
+
+          WARNING:
+          The PrintConfig.dll (%windir%\system32\driverstor\filerepository\prnms003*) on the target host
+          will be overwritten when the exploit runs.
+
+          This module has been tested against Windows 10 Pro x64.
         },
-      'DisclosureDate' => '2018-08-27',
-      'DefaultTarget'  => 0,
-    ))
-
-    register_options([OptString.new('PROCESS',
-        [false, 'Name of process to spawn and inject dll into.', nil])
+        'License' => MSF_LICENSE,
+        'Author' =>
+          [
+            'SandboxEscaper',       # Original discovery and PoC
+            'bwatters-r7',          # msf module
+            'asoto-r7',             # msf module
+            'Jacob Robles'          # msf module
+          ],
+        'Platform' => 'win',
+        'SessionTypes' => ['meterpreter'],
+        'Targets' =>
+          [
+            ['Windows 10 x64', { 'Arch' => ARCH_X64 }]
+          ],
+        'References' =>
+          [
+            ['CVE', '2018-8440'],
+            ['URL', 'https://github.com/SandboxEscaper/randomrepo/'],
+          ],
+        'Notes' =>
+          {
+            # Exploit overwrites PrintConfig.dll, which makes it unusable.
+            'Stability' => [ OS_RESOURCE_LOSS ],
+            'Reliability' => [ REPEATABLE_SESSION ]
+          },
+        'DisclosureDate' => '2018-08-27',
+        'DefaultTarget' => 0
+      )
+    )
+
+    register_options([
+      OptString.new('PROCESS',
+                    [false, 'Name of process to spawn and inject dll into.', nil])
     ])
```

Comment:
It doesn't look like this option is used any more.

Reply:
Woops, good callout, I'll resolve this so long.

Reply:
Resolved with https://github.com/rapid7/metasploit-framework/pull/15384/commits/4ab1b6ca11cc7b025da6094ec4552eaaa8ad657b